### PR TITLE
Feature Field

### PR DESCRIPTION
**Pull request by williamshen-nz**
Feature Field Integration with Nerfstudio

- [x] Need to setup dataset zips on Google Drive
    - [x] Compress to JPG or lossy PNG. Note nerfstudio uses low quality jpg which significantly reduces quality
    - [x] Panda dataset
    - [x] Rooms dataset
- [x] Get image and metrics in model implementation (can ignore)
- [x] Check wandb works
- [x] Try load from checkpoint
- [x] Expose feature field args in model
- [x] Check details against paper
- [x] Get someone to try setting up repo based on README
    - [x] There are issues with CUDA toolkit, nerfstudio local bin conflicts in PATH

### CODE REVIEW

**Comment by geyang** [comment]
Looking now

**Comment by geyang** [comment]
- for scripts, it is common to use the script folder, but it tend to end up polluting the global space. It also tend to cause issues when the same module is installed in multiple conda environments. I suggest changing it to `f3rm_scripts`. 

**Update**: I was on `main`. This branch fixes that problem!

**Comment by williamshen-nz** [comment]
The scripts were originally demo scripts which wouldn't have bee installed with the package. Think it's better on this branch in `f3rm.scripts`

**Comment by geyang** [comment]
LGTM. Some results below:
it is cool that our feature can pick up "air purifier"
Screenshot 2023-09-11 at 3 26 19 PMScreenshot 2023-09-11 at 3 25 56 PM

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,8 @@ __pycache__/
 **/demo_extract_features.png
 **/demo_clip_features*.png
 
+# Datasets
+datasets/
```
Comment (geyang) [inline .gitignore:14]: I would avoid saving data inside code bases even with gitignore. Better to save in a centralized location.

File: README.md [hunk 2]
```diff
@@ -1,54 +1,234 @@
...
+```
+
+#### 3. Clone and install F3RM
+
+```bash
+git clone https://github.com/f3rm/f3rm.git
+cd f3rm
+
+# Install F3RM and its dependencies
 pip install -e .
```
Comment (geyang) [inline README.md:78]: alternative is to add an installation option to install from git.

**Review by geyang — APPROVED** [review]
Looks good to me.

File: .gitignore (hunk 1 above)
Comment (williamshen-nz) [reply .gitignore:14]: Discussed on Slack, user can select where to save themselves

File: README.md (hunk 2 above)
Comment (williamshen-nz) [reply README.md:78]: When the robot stuff is implemented, this codebase will get more complicated and I don't think installing from git would be as suitable.

**Pull request merged**